packages/dashboard/docker-compose.run.yml

```diff
@@ -13,6 +15,11 @@ services:
       - ETHEREUM_RPC_URL=${ETHEREUM_RPC_URL}
       - ETHEREUM_PROVIDER=${ETHEREUM_PROVIDER}
       - ETHEREUM_MNEMONIC=${ETHEREUM_MNEMONIC}
-      - SIDETREE_METHOD=${SIDETREE_METHOD}
+      - S3_BUCKET_NAME=${S3_BUCKET_NAME}
```

Comment:
feels like we should be able to clean this up eventually, perhaps better prefixing would help here.

Reply:
Yeah maybe prefixing with photon or element would be better here